### DISCUSSION TITLE
how to add some querynodes  after installing milvus-cluster by docker-compose?

### DISCUSSION THREAD

**Discussion by houxingxing** [milvus-io/milvus#16397] [Q&A and General discussion]
After starting milvus, I can find only one querynode. 
![image](https://user-images.githubusercontent.com/11939596/161905214-75138acc-811d-4c39-b386-2506caf20d10.png)

IIn general, the query time is about 150ms. When there are too many queries.，sometimes the query time exceeds 5000ms, so I want to add some nodes to alleviate this situation.  could Anyone tell me how to  add some querynodes?
 
I can only find a file in document, which is condected by helm.
[https://milvus.io/cn/docs/v2.0.x/scaleout.md#Scaling-out-1](url)

`
milvus: 2.0.1

`
 
thanks

**Comment by xiaofan-luan** [comment]
Hi @houxingxing ,that might be what you are looking for https://milvus.io/docs/v2.0.x/scaleout.md

**Reply by houxingxing** [reply]
There is not a k8s in my production environment.  I try to modify the docker-compose.yml  file.  like this
![image](https://user-images.githubusercontent.com/11939596/162662879-28f1f3bc-e03a-4472-9b50-517730826bfd.png)
 is this a correct way?

**Reply by houxingxing** [reply]
@xiaofan-luan
